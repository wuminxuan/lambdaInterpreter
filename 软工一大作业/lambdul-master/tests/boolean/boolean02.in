_NOT := \f.(\x.(\y.((f y) x)))
_TRUE := \x.(\y.(x))
_FALSE := \x.(\y.(y))
(_FALSE)
(_TRUE)
(_NOT _FALSE)
(_NOT (_NOT _FALSE))
(_NOT _TRUE)
(_NOT (_NOT _TRUE))
(_NOT)
