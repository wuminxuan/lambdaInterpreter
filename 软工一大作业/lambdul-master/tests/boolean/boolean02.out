_NOT := λf.(λx.(λy.(((f y) x))))
_TRUE := λx.(λy.(x))
_FALSE := λx.(λy.(y))
λx.(λy.(y))
λx.(λy.(x))
λx.(λy.(x))
λx.(λy.(y))
λx.(λy.(y))
λx.(λy.(x))
λf.(λx.(λy.(((f y) x))))
